v3/util/time.go

```diff
@@ -66,7 +66,8 @@ var (
 	CABFBRs_1_6_9_Date                               = time.Date(2020, time.March, 27, 0, 0, 0, 0, time.UTC)
 	CABFBRs_1_7_1_Date                               = time.Date(2020, time.August, 20, 0, 0, 0, 0, time.UTC)
 	AppleReducedLifetimeDate                         = time.Date(2020, time.September, 1, 0, 0, 0, 0, time.UTC)
-	CABFBRs_1_8_0_Date                               = time.Date(2021, time.August, 21, 0, 0, 0, 0, time.UTC)
+     CABFBRs_1_7_9_Date                               = time.Date(2021, time.August, 16, 0, 0, 0, 0, time.UTC)
```

Comment:
```suggestion
	CABFBRs_1_7_9_Date                               = time.Date(2021, time.August, 16, 0, 0, 0, 0, time.UTC)
```

Of course, they're tabs.